src/Utilities.cmake

```diff
@@ -124,3 +124,36 @@ function(detect_architecture arch)
     set(${arch} x64 PARENT_SCOPE)
   endif()
 endfunction()
+
+# detect custom linker
+function(detect_custom_linker LINKER)
+  set(SUPPORTS_LLD OFF)
+  set(SUPPORTS_GOLD OFF)
+  set(SUPPORTS_MOLD OFF)
+  if((CMAKE_CXX_COMPILER_ID MATCHES ".*Clang.*") AND NOT WIN32)
```

Comment:
As far as I have tested, `lld` is supported on Windows. Why is this excluding Windows?